tests-e2e/core/managed_environment_test.go

```diff
@@ -1,1189 +0,0 @@
-package core
```

Comment:
The goal of GITOPSRVCE-540 was actually only to move the `Environment E2E tests` part of this file, into `environment_test.go`. The remaining tests can thus stay within `managed_environment_test.go`.

The overall goal is:
- tests primarily for ManagedEnvironments go in `managed_environment_test.go`
- tests primarily for (AppStudio) Environment go in `environment_test.go`